celesta-maven-plugin/src/test/java/ru/curs/celesta/plugin/maven/GenTestScoreResourcesMojoTest.java

```diff
@@ -52,8 +54,8 @@ public void testExecuteGenScoreResources() throws Exception {
         mojo.execute();
 
         List<String> grainPaths = Arrays.asList(
-            "data/table/table.sql",
-            "seq/sequence.sql"
```

Comment:
This code should be fine as well, and the variant with Stream.of is less readable IMHO.

Reply:
I didn't quite get you. This code is _not_ fine: `assertEquals` below fails because of `\` vs `/` mismatch. We cannot leave it as is, because unit tests fail on Windows.

Reply:
Then `assertEquals(grainPaths, generatedGrainPaths);` is confusing to me. I understood that all paths generated in **score.files** would contain only straight slashes - `'/'` regardless of the OS.
If so the logic that generates such paths should be fixed but not tests.